internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1315,7 +1315,7 @@ func clusterValidated(clusterInstall *hiveext.AgentClusterInstall, status string
 	if err == nil {
 		for _, vRes := range validationRes {
 			for _, v := range vRes {
-				if v.Status == cluster.ValidationFailure {
+				if v.Status != cluster.ValidationSuccess {
```

Comment:
I don't think that we want to include disabled validations.

Reply:
There is no `disabled` type for cluster validations